script/generate-docs

```diff
@@ -89,3 +89,8 @@ generate_markdown_from_json_schema 'Custom Field Answer (String)' 'custom_field_
 generate_markdown_from_json_schema 'Custom Field Answer (Select)' 'custom_field_answer_select.json' '' 'custom_field_answer_select_errors.md not_found.md' > 'custom_field_answer_select.md'
 generate_markdown_from_json_schema 'PDF Manager Template' 'pdf_manager_template.json' '' 'not_found.md' > 'pdf_manager_template.md'
 generate_markdown_from_json_schema 'PDF Manager Batch' 'pdf_manager_batch.json' '' 'pdf_manager_batch_errors.md not_found.md' > 'pdf_manager_batch.md'
+generate_markdown_from_json_schema 'Export/Report' 'export.json' '' 'not_found.md' > 'export.md'
+generate_markdown_from_json_schema 'Export/Report' 'export_by_user_identity.json' '' 'not_found.md' > 'export_by_user_identity.md'
+generate_markdown_from_json_schema 'Export/Report Files check status in batch' 'export_files_check_batch.json' '' 'not_found.md' > 'export_files_check_batch.md'
+generate_markdown_from_json_schema 'Export/Report Files check status' 'export_files_check.json' '' 'not_found.md' > 'export_files_check.md'
+generate_markdown_from_json_schema 'Export/Report Files initiate a run' 'export_files.json' '' 'not_found.md' > 'export_files.md'
```

Comment:
They're not combined because `check status` is user scoped while `initiate a run` is user_identity scoped. So the URLs are different and I though it would be better not to have different routes on the same page even if they're related.